### ISSUE TITLE
Visual feedback when updating subscriptions manually.

### ISSUE THREAD

**Issue by GlenConway** [bstrdsmkr/1Channel#188]
Would it be possible to have some sort of feedback when updating the subscriptions manually?  You don't know it is doing anything until the end and in our house it is pressed multiple times.  This used to be there but I suspect it was taken out around the same time the ability to run automatically at start was added (?).

**Comment by tknorris** [comment]
Yeah, we could add a background dialog to show it. I'll look at this for the next release.
